packages/shared/components/InfoTextList.tsx

```diff
@@ -25,9 +25,21 @@ const contentTextStyles: TextProps = {
   fontSize: 'sm',
 };
 
-export default function InfoTextList({ infoTextItems }: InfoTextListProps) {
+export default function InfoTextList({
+  infoTextItems,
+  children,
+  ...props
+}: InfoTextListProps) {
   return (
-    <Box py={6} borderBottom="1px solid" borderColor="gray.200">
+    <Flex
+      py={6}
+      px={4}
+      flexDir="column"
+      gap="2px"
+      borderBottom="1px solid"
+      borderColor="gray.200"
+      {...props}
+    >
```

Comment:
```suggestion
    <VStack
      py={6}
      px={4}
      spacing="2px"
      borderBottom="1px solid"
      borderColor="gray.200"
      {...props}
    >
```

혹시 이 코드가 작동할지 한 번 테스트 부탁드려도 될까요?? 🤔